library/cohesity_view.py

```diff
@@ -0,0 +1,384 @@
+# !/usr/bin/python
+# Copyright (c) 2019 Cohesity Inc
+# Apache License Version 2.0
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils.basic import AnsibleModule
+from cohesity_management_sdk.cohesity_client import CohesityClient
+from cohesity_management_sdk.exceptions.api_exception import APIException
+from cohesity_management_sdk.models.create_view_request import CreateViewRequest
+from cohesity_management_sdk.models.nfs_root_permissions import NfsRootPermissions
+from cohesity_management_sdk.models.qo_s import QoS
+from cohesity_management_sdk.models.quota_policy import QuotaPolicy
+from cohesity_management_sdk.models.storage_policy_override import StoragePolicyOverride
+from cohesity_management_sdk.models.subnet import Subnet
+from cohesity_management_sdk.models.update_view_param import UpdateViewParam
+
+try:
+    from module_utils.storage.cohesity.cohesity_utilities import cohesity_common_argument_spec,\
+        raise__cohesity_exception__handler
+except Exception:
+    from ansible.module_utils.storage.cohesity.cohesity_utilities import cohesity_common_argument_spec,\
+        raise__cohesity_exception__handler
+
+
+cohesity_client = None
+TWENTY_GB_IN_BYTES = 21474836480
```

Comment:
Use python lib if ansible has it called units you can do 1024 ** units.GiB

Reply:
Name it TWENTY_GB , similarly below

Reply:
ok